meta-hpe/meta-gxp/recipes-connectivity/virtualnic/files/virtualnic

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+if [ $# -ne 1 ]
+then
+action="start"
+else
+action=$1
+fi
+
+gadget_name=obmc_virtualnic
+gadget_dir=/sys/kernel/config/usb_gadget/$gadget_name
+
+set -ex
+
+case "$action" in
+start)
+    (
+    if [ ! -d $gadget_dir ]
+    then
+    mkdir -p $gadget_dir
+	cd /sys/kernel/config/usb_gadget
+	mkdir -p $gadget_name
+	cd $gadget_name
+        echo 0x0419 > bcdDevice                             
+        echo 0x0200 > bcdUSB                                
+
+        echo 0x1d6b > idProduct                             
+        echo 0x0104 > idVendor                              
+
+	echo 0x02 > bDeviceClass
+	echo 0x0C > bDeviceSubClass
+
+
+	mkdir strings/0x409
```

Comment:
Agree. Something weird because the driver on the host side is able to recognize the Mutifunction Composite Gadget even while being swapped. I will test it a little bit deeper

Reply:
Fixed in 3510ac00da9509dbe98a598c609db1a00f447ee5